.github/BOTMETA.yml

```
@@ -1116,6 +1118,12 @@ files:
   $plugins/doc_fragments/xenserver.py:
     maintainers: bvitnik
   $plugins/doc_fragments/hpe3par.py: *hpe3par
+  $plugins/doc_fragments/oracle.py: $team_oracle
```

You can use a glob here. Also, please mark this as community supported:
```suggestion
  $plugins/doc_fragments/oracle*: $team_oracle
    maintainers: $team_oracle
    support: community
```